libraries/disp/viewers/channelselectionview.cpp

```diff
@@ -410,7 +410,7 @@ void ChannelSelectionView::updateDataView()
                 i.remove();
             }
         }
-        SelectionSceneItem* test = static_cast<SelectionSceneItem*>(visibleItemList.first());
+        //SelectionSceneItem* test = static_cast<SelectionSceneItem*>(visibleItemList.first());
```

Comment:
If this variable is no longer needed, we should consider removing it.

Reply:
undone

Reply:
I think that there is a "better" way.